### PR TITLE
documentation: Update documentation to install a recent stack version.

### LINKED ISSUE #272: Exercises do not come with .cabal files
Although the haskell exercises come with `package.yaml` files, they don't come with `.cabal` files, leading to this delightful initial experience if using an old version of `stack`:

```
$ exercism fetch haskell

Not Submitted: 1 problem
haskell (Leap) /home/endgame/exercism/haskell/leap

New:           1 problem
haskell (Leap) /home/endgame/exercism/haskell/leap

unchanged: 0, updated: 0, new: 1

endgame@cyberdwarf:/home/endgame/exercism$ cd haskell/leap/
endgame@cyberdwarf:/home/endgame/exercism/haskell/leap$ stack test
No .cabal file found in directory /home/endgame/exercism/haskell/leap/
```

It might be worth documenting this: a `stack upgrade` followed by a `hash -r` (or spawning a new shell) fixed the problem for me.

### PR DESCRIPTION
**Pull request by rbasso**
Update INSTALLATION.md to mention the need of a recent Stack version.

Closes #272.

### CODE REVIEW

File: docs/INSTALLATION.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-installed it correctly by running this command in a terminal:
+for your operating system. After completing the installation, test that you
+have installed it correctly by running this command in a terminal:
 
 ```bash
 stack --version
 ```
 
-If it outputs Stack's version, then you have it installed correctly.
+If it outputs a stack version equal or greater than 1.1.2, then you have it
```
Comment (petertseng) [inline docs/INSTALLATION.md]: I would say add "to" so it becomes "equal to or greater than", but I guess the rules are a bit lax here.

**Comment by petertseng** [comment]
Intentional to no longer capitalise "stack"?

I'm not terribly picky so I'd say :+1:  either way, maybe someone cares though.

File: docs/INSTALLATION.md (hunk 1 above)
Comment (rbasso) [inline docs/INSTALLATION.md]: Thanks!

**Comment by petertseng** [comment]
:+1:

**Pull request merged**